packages/terra-button/src/Button.jsx

```diff
@@ -255,6 +269,7 @@ class Button extends React.Component {
         { compact: isCompact },
         { 'is-active': this.state.active && !isDisabled },
         { 'is-focused': this.state.focused && !isDisabled },
+        { 'is-selected': customProps.isSelectable && this.state.isSelected && !isDisabled },
```

Comment:
let's spread out the isSelectable on top and use it instead of referring it through customProps chaining on multiple place for better readablity

Reply:
That would cause unlisted prop lint err. Instead now moved to the top like this: https://github.com/cerner/terra-core/pull/3940/commits/54c4c362b16f27f2bdfc0acb91705ac1e11d9c7e#diff-39f6c87ccf92298263c905de24e815f96ac1c87e46309e7d88ac4513f8bb7193R132 

This way handlers can use it outside of render.